services/managers/srp_manager.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+
+
+import json
+import urllib2
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+class srpManager():
+    @staticmethod
+    def get_kill_id (killboard_link):
+        str = (killboard_link)
+        set = '0123456789'
+        kill_id = ''.join([c for c in str if c in set])

```

Comment:
`kill_id = re.match(re.compile(r'^\d+'), str)` or something - this would pull out only numbers from the url